docs/user-guide/expressions/plugins.md

```diff
@@ -94,48 +94,18 @@ expression in batches. Whereas for other operations this would not be allowed, t
 # expression_lib/__init__.py
 import polars as pl
 from polars.type_aliases import IntoExpr
-from polars.utils.udfs import _get_shared_lib_location
+from polars.plugins import register_plugin
 
-from expression_lib.utils import parse_into_expr
-
-# Boilerplate needed to inform Polars of the location of binary wheel.
-lib = _get_shared_lib_location(__file__)
 
 def pig_latinnify(expr: IntoExpr, capitalize: bool = False) -> pl.Expr:
-    expr = parse_into_expr(expr)
-    return expr.register_plugin(
-        lib=lib,
+    return register_plugin(
+        expr,
+        plugin_location=plugin_location,
```

Comment:
`plugin_location` is not defined in this code example.

I would add a line before this function with `plugin_locations = Path(__file__).parent` here.

Reply:
🤦 embarrassed to have missed this, I shouldn't be making such mistakes anymore. thanks for the catch!

Reply:
Haha, I don't think those kinds of mistakes is something you can grow out of. Everyone makes them :) which is why we have code reviews :ok_hand: